src/hooks/useKnockoutExplorer.ts

```diff
@@ -364,11 +364,16 @@ function updateContextsFromPortalMessage(inputs: DataExplorerInputsFrame) {
     if (inputs.flights.indexOf(Flights.PKPartitionKeyTest) !== -1) {
       userContext.features.partitionKeyDefault2 = true;
     }
-    if (inputs.flights.indexOf(Flights.PhoenixNotebooks) !== -1) {
+    if (configContext.isPhoenixEnabled === true) {
```

Comment:
why have this check here? Keep this based only on flights.

set userContext.features.phoenixNotebooks in configCOntext similar to how we use isTerminalEnabled

Reply:
Instead updating base of configContext